examples/askui-nodejs-base-example/package.json

```diff
@@ -7,17 +7,18 @@
         "lint:staged": "lint-staged"
     },
     "dependencies": {
-        "askui": "0.1.7"
+        "askui": "0.1.7",
```

Comment:
We don't have a version `0.1.7`, yet.

Reply:
the reason why I upgraded the version, was that I already updated the example with the current changes, and the current version doesn't support that version

Reply:
I see but I don't like that it is pointing to version that doesn't exist. No one can actually run that example right now. What about pointing to `../../packages/askui-nodejs` for the time being.

Reply:
https://askyourui.atlassian.net/browse/AS-1081

Reply:
I would prefer then to rename it to the e2e test and point it to the `../../packages/askui-nodejs`. 
because it can't be made sure that it's up to date without first releasing a new version.
maybe also exclude updating it from the tickets. then we can keep the old version

Reply:
I agree with the last point. I would also go for keeping the old version and only upgrading examples after a new release. I think this would save us a lot of trouble.

Because of that I would see `../../packages/askui` as a temporary fix that we are going to replace with a proper version after the next release today. So I don't think renaming is necessary.

Reply:
the main drawback, if I want an example for a specific version won't be included inside the code when I check out the same tag number as the version

Reply:
Let's discuss it later after the Daily.